inference-engine/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -55,6 +55,11 @@ if(NGRAPH_ONNX_IMPORT_ENABLE)
     add_dependencies(${TARGET_NAME} inference_engine_onnx_reader)
 endif()
 
+if(NGRAPH_PDPD_FRONTEND_ENABLE)
+    target_compile_definitions(${TARGET_NAME} PRIVATE
```

Comment:
Can we reuse the approach which is used in nGraph unit tests for PDPD Frontend?
CMake generates models and these models are using in tests.
Can we reuse these models?

Reply:
It looks like overkill for just 2 models and onnx have them static in the same way